### ISSUE TITLE
Unconditional image generation from Muddit.

### ISSUE THREAD

**Issue by basusattwik** [M-E-AGI-Lab/Muddit#10]
Could you please clarify if it is possible to generate unconditional image samples from Muddit? Is it possible to set CFG guidance scale as 0.0 and pass in null conditions? Thank you!

**Comment by viiika** [comment]
Thanks for reading Muddit! Glad you found it interesting. Muddit is built on Meissonic [ICLR 2025], feel free to take a look at https://github.com/viiika/Meissonic.

Regarding unconditional generation: the released model was not explicitly trained with classifier-free condition dropout or null conditioning. Therefore, strictly speaking, we would not describe it as a model trained for true unconditional generation. That said, at inference time, you may try using an prompt (e.g., " ") as an approximation. For more details, please refer to the training script in the Meissonic repo.

We currently provide the checkpoints in safetensors format (https://huggingface.co/MeissonFlow), which is PyTorch-compatible and can be loaded directly in PyTorch / Hugging Face.
